documentation/src/main/asciidoc/stories/assembly_cache_configuration.adoc

```diff
@@ -10,6 +10,19 @@ For instance, you can configure {brandname} clusters to use replicated caches or
 Your configuration also defines the characteristics of your caches and enables the {brandname} capabilities that you want to use when handling data.
 For instance, you can configure how {brandname} encodes entries in your caches, whether replication requests happen synchronously or asynchronously between nodes, if entries are mortal or immortal, and so on.
 
+[IMPORTANT]
+====
+{brandname} places a restriction of less than or equal to 255 characters on a cache name or a cache template name.
+If you exceed this character limit, {brandname} throws an exception message
```

Comment:
Just a really small thing, the same in the other file.

```suggestion
If you exceed this character limit, {brandname} throws an exception.
```

Reply:
Updated.